docs/development/core/development-functional-tests.asciidoc

```diff
@@ -133,6 +141,28 @@ The `FunctionalTestRunner`'s primary purpose is to execute test files. These fil
 **Test Suite**:::
 A test suite is a collection of tests defined by calling `describe()`, and then populated with tests and setup/teardown hooks by calling `it()`, `before()`, `beforeEach()`, etc. Every test file must define only one top level test suite, and test suites can have as many nested test suites as they like.
 
+**Cross-browser testing**:::
+All the functional tests are executed in Chrome by default. To run specific tests in both Chrome and Firefox, `smoke` tag should be assigned:
+
+["source","js"]
+-----------
+// tags must be defined in `describe` function
```

Comment:
This is the part I would recommend be put in section about 'tags' maybe before this heading, I know I tried using tags in 'it' block and it did not work.  Also maybe clarify how tags are added and which tags exist.  I know there is a file structure for it, just some basic explanation.